drivers/usb/device/usb_dc_b91.c

```diff
@@ -1726,7 +1726,7 @@ static void usbd_work_handler(struct k_work *item)
 	}
 }
 
-static int usb_init(const struct device *arg)
+static int usb_init(void)
```

Comment:
See the next
`SYS_INIT(usb_init, POST_KERNEL, CONFIG_KERNEL_INIT_PRIORITY_DEVICE);`

which finally leads to
```
struct init_entry {
	/**
	 * Initialization function for the init entry.
	 *
	 * @param dev Device pointer, NULL if not a device init function.
	 *
	 * @retval 0 On success
	 * @retval -errno If init fails.
	 */
	int (*init)(const struct device *dev);
	/**
	 * If the init entry belongs to a device, this fields stores a
	 * reference to it, otherwise it is set to NULL.
	 */
	const struct device *dev;
};
```
As you see in Zephyr init function prototype is the next
`int (*init)(const struct device *dev);`
And we should keep this convention.